### PR TITLE
 MSGraphUser  restore a recently deleted user

### PR DESCRIPTION
**Pull request by sudkumar1**
Added a `msgraph-user-restore-user`  to restore a recently deleted  user

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: #17418

## Description
Added a msgraph-user-restore-user command to restore the recently deleted user.

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [x] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

File: Packs/MicrosoftGraphUser/ReleaseNotes/1_5_0.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Azure Active Directory Users
+- Added the ***msgraph-user-restore-user*** command which restore the user from recently delted user.
```
Comment (israelpoli) [inline Packs/MicrosoftGraphUser/ReleaseNotes/1_5_0.md:4]:
```suggestion
- Added the ***msgraph-user-restore-user*** command which restore the user from recently deleted user.
```

File: Packs/MicrosoftGraphUser/Integrations/MicrosoftGraphUser/MicrosoftGraphUser.yml [hunk 2]
```diff
@@ -3,613 +3,681 @@ commonfields:
...
+        Permission: Directory.AccessAsUser.All(Delegated)
+      execution: false
+      name: msgraph-user-session-revoke
+      outputs: []
+    - name: msgraph-user-restore-user
+      description: Restores a recently deleted user from deleted items.
+      deprecated: false
+      arguments:
+        - name: user_id
+          description: Object ID of the delted user that needs to be restored
```
Comment (israelpoli) [inline Packs/MicrosoftGraphUser/Integrations/MicrosoftGraphUser/MicrosoftGraphUser.yml:659]:
```suggestion
          description: Object ID of the deleted user that needs to be restored
```

File: Packs/MicrosoftGraphUser/Integrations/MicrosoftGraphUser/MicrosoftGraphUser.py [hunk 3]
```diff
@@ -467,6 +486,8 @@ def main():
         'msgraph-user-get-manager': get_manager_command,
         'msgraph-user-assign-manager': assign_manager_command,
         'msgraph-user-session-revoke': revoke_user_session_command,
+        'msgraph-user-restore-user': restore_user_command,
+
```
Comment (israelpoli) [inline Packs/MicrosoftGraphUser/Integrations/MicrosoftGraphUser/MicrosoftGraphUser.py:490]: Please remove this empty line

**Review by israelpoli — CHANGES_REQUESTED** [review]
Hi @sudkumar1 
Thank you for your contribution

Please see my comments below
I will review the readme file later today.

Thanks

**Comment by bakatzir** [comment]
Hey team, what is the status of this PR?

**Comment by sudkumar1** [comment]
hi @israelpoli  I have committed  the suggested changes, please let me know if  any other changes are required

**Comment by israelpoli** [comment]
Hi @sudkumar1 

Sorry for the long delay, this PR eluded me,
can you please merge from master so I continue from here?

Thanx

**Comment by israelpoli** [comment]
Closed due to inactivity.

**Pull request closed without merging**